### PR TITLE
kodi: enable cross compilation for 32bit arm architectures

### PR DESCRIPTION
**Pull request by marmeladema**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by marmeladema** [comment]
Hopefully this should help provide Kodi 19 for raspberry pie boards

File: srcpkgs/kodi/template [hunk 1]
```diff
@@ -77,11 +77,10 @@ _gtest_filter+=":TestWebServer.CanGetRangedFileRangeFirstSecond"
...
 case "$XBPS_TARGET_MACHINE" in
-	aarch64*)
-		hostmakedepends+=" libmariadbclient-devel SDL2_image-devel lzo-devel"
-		hostmakedepends+=" libwaylandpp-devel"
-		makedepends+=" python3-devel ffmpeg-devel"
-		configure_args+=" -DENABLE_VAAPI=OFF"
+	armv6l*)
+		makedepends+=" libatomic-devel"
+		configure_args+=" -DCMAKE_EXE_LINKER_FLAGS=-latomic"
+		LDFLAGS+=" -latomic"
```
Comment (ericonr) [inline srcpkgs/kodi/template:83]: This should be a conditional using `XBPS_TARGET_NO_ATOMIC8`

File: srcpkgs/kodi/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'kodi'
 pkgname=kodi
 version=19.0
-revision=1
+revision=2
```
Comment (ericonr) [inline srcpkgs/kodi/template:4]: It's just adding new supported archs, so revbump isn't necessary.

**Comment by marmeladema** [comment]
@ericonr updated

File: srcpkgs/kodi/template [hunk 3]
```diff
@@ -98,6 +92,16 @@ esac
 
 if [ -z "$CROSS_BUILD" ]; then
 	make_build_target+=" kodi-test"
+else
+	hostmakedepends+=" libmariadbclient-devel SDL2_image-devel lzo-devel"
+	hostmakedepends+=" libwaylandpp-devel"
+	makedepends+=" python3-devel ffmpeg-devel"
```
Comment (ericonr) [inline srcpkgs/kodi/template:98]: Add `python3-devel` unconditionally to `makedepends`, `ffmpeg-devel` is there already.

File: srcpkgs/kodi/template [hunk 4]
```diff
@@ -98,6 +92,16 @@ esac
 
 if [ -z "$CROSS_BUILD" ]; then
 	make_build_target+=" kodi-test"
+else
+	hostmakedepends+=" libmariadbclient-devel SDL2_image-devel lzo-devel"
+	hostmakedepends+=" libwaylandpp-devel"
```
Comment (ericonr) [inline srcpkgs/kodi/template:97]:
```suggestion
	hostmakedepends+=" libmariadbclient-devel SDL2_image-devel lzo-devel
	 libwaylandpp-devel"
```

File: srcpkgs/kodi/template (hunk 3 above)
Comment (marmeladema) [reply srcpkgs/kodi/template]: Done

File: srcpkgs/kodi/template (hunk 4 above)
Comment (marmeladema) [reply srcpkgs/kodi/template]: Done

**Pull request merged**